internal/bzlmod/default_gazelle_overrides.bzl

```diff
@@ -57,7 +63,8 @@ DEFAULT_DIRECTIVES_BY_PATH = {
         "gazelle:proto disable",
     ],
     "k8s.io/apimachinery": [
-        "gazelle:proto disable",
+        "gazelle:go_generate_proto false",
+        "gazelle:proto_import_prefix k8s.io/apimachinery",
```

Comment:
This should be quite stable, because any change to proto import prefix requires changing import statements in proto files